integrationTests/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": "../base.tsconfig.json",
+    // NOTE: Everything except what you want compiled into JS should be in the exclusions.
+    "compilerOptions": {
+        "outDir": "./dist"
+    },
+    "exclude": [
+        "node_modules",
+        "dist"
+    ],
+    "include": [
+        "./renderedTemplates"
```

Comment:
This is specific to the core package, it's not needed here. And I don't think compilerOptions and exclude are needed either, because nothing is ever compiled here.

Reply:
Aren't the .ts files transpiled and use these options?